apps/site/src/components/dom/Layout.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { ReactNode, useRef } from "react";
+import dynamic from "next/dynamic";
+const Scene = dynamic(() => import("@/components/canvas/Scene"), {
+	ssr: false,
+});
```

Comment:
Suggestion: I don't think you need to import `Scene` dynamically (and disable SSR) here because both are client components. I tried importing it normally and it still works.

Reply:
I made the change, but let's put a pin in this just in case. I am unsure about how Three.js / React Three Fiber interacts with SSR, which I believe this dynamic import prevents. Just a learning from using Framer Motion, which had odd behavior with SSR.